apps/masterbots.ai/components/routes/chat/chat.tsx

```
@@ -162,39 +185,47 @@ export function Chat({
       console.error("User is not logged in or session expired.");
       toast.error("Failed to start conversation. Please reload and try again.");
       return;
     }
 
-    // * Loading: processing your request + opening pop-up...
-    setLoadingState("processing");
-    setIsOpenPopup(true);
+    if (isNewChat) {
+      const optimisticThread: Thread = {
+        threadId,
+        chatbotId: chatbot!.chatbotId,
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)